eureka2-integration/src/test/java/com/netflix/eureka2/integration/ReadWriteClusterIntegrationTest.java

```diff
@@ -29,13 +31,17 @@
 public class ReadWriteClusterIntegrationTest {
 
     @Rule
-    public final EurekaDeploymentResource eurekaDeploymentResource = new EurekaDeploymentResource(3, 6);
+    public final EurekaDeploymentResource eurekaDeploymentResource = new EurekaDeploymentResource(1, 1);
```

Comment:
My mistake. I did some testing and I wanted to simplify configuration. I will revert this change.